sensor_msgs/PointCloud2 cloud
string filepath
---
